text/src/autogluon/text/automm/configs/environment/default.yaml

```diff
@@ -0,0 +1,12 @@
+env:
+  num_gpus: -1
+  num_nodes: 1
+  batch_size: 128  # this is a desired batch size; pl trainer will accumulate gradients when per step batch is smaller.
+  per_gpu_batch_size: 16  # you can define this according to your GPU memory
+  precision: 16  # training precision
+  num_workers: 2  # pytorch training dataloader workers
+  num_workers_evaluation: 2  # pytorch prediction/test dataloader workers
```

Comment:
This is a because we use "ddp_spawn" as default, which can only use limited cpu workers. Otherwise, the job would get stucked. Refer to: https://pytorch-lightning.readthedocs.io/en/stable/advanced/multi_gpu.html#distributed-data-parallel-spawn

The reason we use "ddp_spawn" instead of "ddp" is to support jupyter notebook user cases. "ddp" seems not working in notebook.